config-usa.yml

```diff
@@ -0,0 +1,125 @@
+# Configuration file for nationwide GraphHopper routers. Vehicle profiles are declared for auto, bike, and ped
```

Comment:
This file is a copy of the nationwide config used in https://github.com/replicahq/graphhopper/pull/20 - the code in that PR is needed to build the GH graph files that this export command requires as input (and both PRs were used to build the nationwide MVP), so it was helpful to have this config in both branches.